nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -660,17 +673,46 @@ public void errorSerializationTest() {
                 "RETURN_NULLABLE",
                 "returning @Nullable expression from method",
                 "com.uber.Super",
-                "test(java.lang.Object)"),
+                "test(java.lang.Object)",
+                "METHOD",
+                "com.uber.Super",
+                "test(java.lang.Object",
```

Comment:
Isn't there a missing `')'` here? And shouldn't the test fail because of it?

Reply:
yes it should fail, updated the test.